app/views/comments/new.html.erb

```diff
@@ -8,7 +8,7 @@
       <div class='col-sm-10 col-sm-offset-1'>
         <%= render partial: 'comments/header' %>
 
-        <%= render partial: 'comments/form', locals: { url: comments_path, cancel_path: after_update_object_path_for(@comment.commentable), submit_text: "Submit" } %>
+        <%= render partial: 'comments/form', locals: { url: comments_path, cancel_path: after_update_object_path_for(@comment.commentable), submit_text: "#{t('comments.submit.submit')}" } %>
```

Comment:
This is very similar to the previous; this would be better named as `comments.new.submit`.